src/views/open-api-docs-view/types.ts

```diff
@@ -83,10 +83,11 @@ export interface OpenApiDocsParams extends ParsedUrlQuery {
  */
 type DividerNavItem = { divider: true }
 type HeadingNavItem = { heading: string }
-type LinkNavItem = {
+export type LinkNavItem = {
 	title: string
 	fullPath: string
 	theme?: ProductSlug
+	isActive?: boolean
```

Comment:
Revised split-out hook has more well-constructed types, this property is optionally present on `LinkNavItem` and that's now properly reflected 👍